src/Service.Host/client/src/components/boards/RevisionTab.js

```diff
@@ -95,7 +87,7 @@ function RevisionTab({
                             <Grid item xs={8} />
                             <Grid item xs={3}>
                                 <Dropdown
-                                    value={revision.splitBom}
+                                    value={revision.splitBom ?? 'No'}
```

Comment:
As above

Reply:
yes to both - good catch. if only I could read!